deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -410,7 +411,22 @@
           "keySource": "Microsoft.Storage"
         },
         "accessTier": "Hot"
-      }
+      },
+      "resources": [
+        {
+          "type": "Microsoft.Storage/storageAccounts/providers/roleAssignments",
+          "apiVersion": "2020-03-01-preview",
+          "name": "[concat(variables('storage_account_name'), '/Microsoft.Authorization/', guid(uniqueString(variables('storage_account_name'))))]",
+          "dependsOn": [
+            "[resourceId('Microsoft.Storage/storageAccounts', variables('storage_account_name'))]"
+
+          ],
+          "properties": {
+            "roleDefinitionId": "[variables('storage_blob_contributor_role')]",
+            "principalId": "[reference(variables('app_service_resource_id'), '2015-08-01', 'Full').Identity.principalId]"
```

Comment:
I don't see a `scope` applied here which would limit this role to a specific storage account. What `scope` does this role get applied to in that case? In other words does the MI have `blob contributor` access to a specific storage account or all storage accounts at the resource group/subscription level?

Reply:
This role assignment is nested under the storage account created above and just has access to that storage account.